apps/question-bank/src/main/java/gov/cdc/nbs/questionbank/entity/WaUiMetadata.java

```diff
@@ -524,6 +525,33 @@ private void setVisible(boolean visible) {
     this.displayInd = visible ? "T" : "F";
   }
 
+  public void update(PageContentCommand.GroupSubsection command) {
+    setBlockNm(command.blockName());
```

Comment:
nit: if you're within the object itself it's better to prefer  assigning the  value to the field directly.  

Just a bit of context: When using this pattern with the commands the entities should not really have setters as that allows the entity state to be changed from anywhere.  The purpose of the `command` is to encapsulate all values needed to carryout a change forcing the business logic to be contained within the entity and not spread around the application.